### DISCUSSION TITLE
[Question]: failed to solve: process "/bin/sh -c apk --no-cache add curl &&     npm ci" did not complete successfully: exit code: 146

### DISCUSSION THREAD

**Discussion by longjiansina** [danny-avila/LibreChat#1662] [Troubleshooting]
### What is your question?

My system information is as follows:
LSB Version:    :core-4.1-amd64:core-4.1-noarch
Distributor ID: CentOS
Description:    CentOS Linux release 7.9.2009 (Core)
Release:        7.9.2009
Codename:       Core

I get an error when executing the following command:
docker-compose up

error message:
![1](https://github.com/danny-avila/LibreChat/assets/129648324/82686882-100e-4d01-a522-6ba5ad3d961d)
![2](https://github.com/danny-avila/LibreChat/assets/129648324/9144ad65-f2b1-4a6d-8b8a-da8042e18713)
failed to solve: process "/bin/sh -c apk --no-cache add curl &&     npm ci" did not complete successfully: exit code: 146

### More Details

。

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment]
you could try using the pre-built image instead

You can edit the `docker-compose.override.yml` file (rename it without the `.example`)

 you need something like this:

```
version: '3.4'

services:
  api:
    image: ghcr.io/danny-avila/librechat-dev:latest
```

see also for more information:
https://docs.librechat.ai/install/configuration/docker_override.html

**Comment by longjiansina** [comment]
ok.yeah!
